html/browsers/windows/embedded-opener-remove-frame.html

```diff
@@ -1,10 +1,36 @@
 <!doctype html>
-<title>opener and "removed" embedded documents</title>
+<title>opener and discarded browsing contexts</title>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <div id=log></div>
 <iframe name=matchesastring></iframe>
 <script>
+function testOpener(otherW, thisW, discardOtherBC) {
+  assert_equals(otherW.opener, thisW, "opener before removal");
+
+  const openerDesc = Object.getOwnPropertyDescriptor(otherW, "opener"),
+        openerGet = openerDesc.get;
+
+  assert_equals(openerGet(), thisW, "opener before removal via directly invoking the getter");
+  discardOtherBC();
+  assert_equals(otherW.opener, null, "opener after removal");
+  assert_equals(openerGet(), null, "opener after removal via directly invoking the getter");
+
+  otherW.opener = null;
```

Comment:
It's a little weird to do this after the opener is already asserted to be null, but I guess we were already testing that.

Reply:
Mainly to ensure the setter doesn't do anything weird. I could remove this if desired...